artifactory/auth/sshlogin.go

```diff
@@ -65,12 +103,38 @@ func sshAuthentication(url string, sshKey, sshPassphrase []byte) (map[string]str
 	if err = json.Unmarshal(buf.Bytes(), &result); errorutils.CheckError(err) != nil {
 		return nil, "", err
 	}
-	url = utils.AddTrailingSlashIfNeeded(result.Href)
+	url := utils.AddTrailingSlashIfNeeded(result.Href)
 	sshAuthHeaders := result.Headers
-	log.Info("SSH authentication successful.")
 	return sshAuthHeaders, url, nil
 }
 
+func readSshKeyAndPassphrase(sshKeyPath, sshPassphrase string) ([]byte, []byte, error) {
+	sshKey, err := ioutil.ReadFile(utils.ReplaceTildeWithUserHome(sshKeyPath))
+	if err != nil {
+		return nil, nil, errorutils.CheckError(err)
+	}
+	if len(sshPassphrase) == 0 {
+		encryptedKey, err := IsEncrypted(sshKey)
+		if err != nil {
+			return nil, nil, errorutils.CheckError(err)
+		}
+		// If key is encrypted but no passphrase specified
+		if encryptedKey {
+			return nil, nil, errorutils.CheckError(errors.New("SSH Key is encrypted but no passphrase was specified. Please pass a passphrase with the --ssh-passphrase flag"))
```

Comment:
This code is not necessarily used by JFrog CLI (it is part of jfrog-client-go, which is a library which can be used by any client).
I therefore suggest removing the second part of the error message:
**...Please pass a passphrase with the --ssh-passphrase flag**